config/application.rb

```diff
@@ -194,7 +194,6 @@ def try_config_for(*args)
     config.three_scale.plan_rules.merge!(try_config_for(:plan_rules) || {})
 
     config.three_scale.features = ActiveSupport::OrderedOptions.new
-    config.three_scale.features.account_deletion = ActiveSupport::OrderedOptions.new
```

Comment:
It is not necessary for it to work 😉 the `AccountDeletionConfig` takes care of everything 👍